taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -31,9 +31,18 @@ def get_components():
     } for (name, project) in build_config['projects'].items()]
 
 
-def get_version():
-    return _read_build_config()["libraryVersion"]
+def get_version(config):
```

Comment:
Looks like only `config.params` is ever used, so might be better to make this function just accept `params`?